### DISCUSSION TITLE
我们可以拥有 开题报告 的 LaTeX 模板吗？

### DISCUSSION THREAD

**Discussion by Freed-Wu** [ustctug/ustcthesis#401] [Ideas]
Can we have a LaTeX template for the proposal?

**Comment by zepinglee** [comment]
有哪些具体的格式要求？

**Reply by Freed-Wu** [reply]
- [word](https://cicpi.ustc.edu.cn/indico/conferenceDisplay.py?confId=971)
- [latex](https://github.com/cgdsss/thesis_proposal_ustc)
- [typst](https://github.com/ustc-ivclab/ustc-proposal)

I try to write a typst, Can we host it on this organization?

You can download [here](https://github.com/ustc-ivclab/ustc-proposal/actions/runs/12902994355/artifacts/2466227994).

**Reply by Freed-Wu** [reply]
@zepinglee

**Reply by zepinglee** [reply]
I've invited you as a org member.
